com.regnosys.rosetta/src/com/regnosys/rosetta/Rosetta.xtext

```diff
@@ -51,35 +59,33 @@ Function:
 	('inputs:' inputs += Attribute+)?
 	('output:' output = Attribute)?
 	
-	(shortcuts    += ShortcutDeclaration ';'?)*
-	conditions    += Condition*
-	operation     = Operation?
-	postConditions+= PostCondition*
+	(shortcuts     += ShortcutDeclaration ';'?)*
+	conditions     += Condition*
+	operation       = Operation?
+	postConditions += PostCondition*
 ;
 
 ShortcutDeclaration:
 	'alias' RosettaNamed ':' Definable?  expression=RosettaCalcExpression
 ;
 
-Attribute:
-	RosettaNamed RosettaTyped card=RosettaCardinality Definable? Annotations?;
 
 Condition:
 	'condition' RosettaNamed? ':' Definable?
 		Annotations?
 		(expressions+=RosettaCalcExpression ';')+
```

Comment:
Is there any way we can remove the need for the `;` here?

Reply:
Not with ´RosettaCalcExpression´ following by ´RosettaCalcExpression´. For `condition` I was thinking about having only one expression.